_maps/map_files/RandomRuins/SpaceRuins/turretedoutpost.dmm

```diff
@@ -70,6 +70,7 @@
 /area/ruin/space/unpowered)
 "q" = (
 /obj/machinery/light,
+/mob/living/simple_animal/hostile/syndicate/melee/autogib,
```

Comment:
Every other ruin uses a subtype of `melee/autogib`, is it intentional that you used the base type here? If so, just disregard this

Reply:
this is correct. in fact, the other non-depot maps using the depot subtype should be looked at at some point because they might be causing the alert level on the depot to raise inadvertently.